Manifest/EndUser/manifest.json

```diff
@@ -10,6 +10,59 @@
     "privacyUrl": "<<privacyUrl>>",
     "termsOfUseUrl": "<<termsOfUseUrl>>"
   },
+  "localizationInfo": {
+    "defaultLanguageTag": "en-US",
+    "additionalLanguages": [
+      {
+        "languageTag": "en-US",
+        "file": "en-US.json"
+      },
+      {
+        "languageTag": "de-DE",
```

Comment:
Why don't we use the neutral language here (de) instead of de-de? Are we intentionally excluding say de-ch?

Reply:
We have updated all languages to neutral culture. Just zh-CN and zh-TW is not changed as they contain different text for two different culture.

Reply:
Pls see comment below about pt-pt and pt-br, as they cannot be lumped together under pt.

Reply:
Hello @aosolis,
We have initially got translated for pt-BR so I have reverted it as pt-BR.